central/image/service/service_impl.go

```diff
@@ -382,6 +382,13 @@ func (s *serviceImpl) EnrichLocalImageInternal(ctx context.Context, request *v1.
 
 	defer s.internalScanSemaphore.Release(1)
 
+	hasErrors := false
+	if request.Error != "" {
+		hasErrors = true
+		log.Infof("received image enrichment request with errors %q: %v", request.GetImageName().GetFullName(), request.GetError())
```

Comment:
similarly, would a warning be good or could that blow up logs?

Reply:
> debug or do we want to keep this around?

Intent was to keep it, will help answer 'why are my scans failing' - today support will search through central logs to find why central initiated scans are failing, this allows them to continue that practice.  Otherwise they would have to know to go to the sensor logs on cluster X to determine why.

Its possible that it could blow up the logs if there are lots of failures yes. In my experience thats better then not having logs and guessing.

> similarly, would a warning be good or could that blow up logs?

Not sure I follow, is the suggestion to change the log level to `warn`? That would make it harder to suppress (if thats the intention) than if at `info` as `warn` is higher level. I would prefer to have more logs than less, is there a reason to log less that I may be missing?

Reply:
will change to warn

Reply:
fixed